services/coderabbit/coderabbit-stats.service.js

```diff
@@ -0,0 +1,69 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  reviews: Joi.number().required(),
+}).required()
+
+class CodeRabbitStats extends BaseJsonService {
+  static category = 'analysis'
+  static route = {
+    base: 'coderabbit',
+    pattern: 'stats/:provider/:org/:repo',
+  }
+
+  static examples = [
+    {
+      title: 'CodeRabbit Review Stats',
+      namedParams: {
+        provider: 'github',
+        org: 'coderabbitai',
+        repo: 'ast-grep-essentials',
+      },
+      staticPreview: this.render({
+        reviews: 101,
+      }),
+      documentation: 'Shows the number of CodeRabbit reviews for a repository',
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'CodeRabbit',
+    labelColor: '171717',
+  }
+
+  static render({ reviews }) {
+    return {
+      message: `${reviews} Reviews`,
+      color: 'ff570a',
+    }
+  }
+
+  static renderError({ message }) {
+    return {
+      message,
+      color: '9f9f9f',
```

Comment:
Can we ditch the custom colours here and stick with the standard palette please.
We can just get rid of the `labelColor` in `defaultBadgeData`. All badges should use the default `#555555` for the label (left side of the badge)

For the message (right side), lets go with `blue`. We use this for informational badges in our palette, like this one where all badges get the same colour regardless of the number.

Reply:
I have removed colors fully